pkgs/servers/web-apps/wiki-js/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, lib, nixosTests }:
+
+stdenv.mkDerivation rec {
+  pname = "wiki-js";
+  version = "2.5.191";
+
+  src = fetchurl {
+    url = "https://github.com/Requarks/wiki/releases/download/${version}/${pname}.tar.gz";
+    sha256 = "sha256-lEHelZTFZxcds7t3TCMcd9b3rKdml54A0/V7gcQIyPA=";
+  };
+
+  buildCommand = ''
+    mkdir $out
+    tar xzvf $src -C $out
```

Comment:
Would it be easier to use fetchzip here?

Reply:
@SuperSandro2000 I updated the package now. Rather than using fetchzip, the creation of `$out` is happening in the `installPhase` now. This has the benefit, that a `patchPhase` is executed and one can - if needed - add custom changes rather trivially with `overrideAttrs`.